plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -243,28 +243,6 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
             active_producers.insert(p.producer_name);
          }
```

Comment:
Also, remove everything else above this line within this function. And remove the member variables `_last_signed_block_time`, `_start_time`, and `_last_signed_block_num` from `producer_plugin_impl`.

Although maybe we still need to keep `_start_time` after correcting the watermark logic, since that effectively acts as the time watermark for all locally configured producers even if an entry doesn't yet exist for them in `_producer_watermarks`.

Reply:
kept the `_start_time` in pending a deeper review.